File: nysol_python/mcmd/methods/convTex/db/smp/mmvsim.py
# ================================================
# コマンドマニュアル自動作成用サンプルコードデータ
# ================================================
db={}
db['name']='mmvsim'

############################### IDATA
db['idatas']=[]

data={}
data['name']='dat1.csv'
data['text']='''
t,x,y
1,14,0.17
2,11,0.2
3,32,0.15
4,13,0.33
5,8,0.1
6,19,0.56
'''
db['idatas'].append(data)

############################### SCRIPTS
db['scripts']=[]

script={}
script['title']='基本例'
script['comment']='''
``x、y`` 項目についてのピアソンの積率相関係数を3期を窓として計算する。
'''
script['sh_code']='''
mmvsim s=t t=3 c=pearson f=x,y a=sim i=dat1.csv o=rsl1.csv
'''
script['py_code']='''
nm.mmvsim(s="t", t="3", c="pearson", f="x,y", a="sim", i="dat1.csv", o="rsl1.csv").run()
'''
script['odatas']='rsl1.csv'
db['scripts'].append(script)

File: olddoc/mcmd/jp/examples/mmvsim.rb
#!/usr/bin/env ruby
# coding: utf-8
require "./mkTex.rb"

File.open("dat1.csv","w"){|fpw| fpw.write(
<<'EOF'
t,x,y
1,14,0.17
2,11,0.2
3,32,0.15
4,13,0.33
5,8,0.1
6,19,0.56
EOF
)}

############## 例1
title="基本例"
comment=<<'EOF'
\verb|x、y|項目についてのピアソンの積率相関係数を3期を窓として計算する。
EOF
scp=<<'EOF'
more dat1.csv
mmvsim s=t t=3 c=pearson f=x,y a=sim i=dat1.csv o=rsl1.csv
more rsl1.csv
EOF
run(scp,title,comment)